crates/shader-crate-template/Cargo.toml

```diff
@@ -15,8 +15,77 @@ spirv-std = { git = "https://github.com/Rust-GPU/rust-gpu", rev = "82a0f69" }
 [target.'cfg(target_arch = "spirv")'.dependencies]
 glam = { version = "0.29", default-features = false, features = ["libm"] }
 
-
 # Dependencies for CPU code
 [target.'cfg(not(target_arch = "spirv"))'.dependencies]
 glam = { version = "0.29", features = ["std"] }
 
+[package.metadata.rust-gpu.build]
+# Where to output the compiled shader. Defaults to where `cargo gpu` is called from.
+# TODO: Should it default to the root of the shader crate?
+output-dir = "./"
+# Set shader crate's cargo default-features
+no_default_features = false
+# Set shader crate's cargo features.  
+features = []
+# The compile target.
+# TODO: `cargo gpu show targets` for all available options.
```

Comment:
Is this still a TODO?

Reply:
Yes, we have an issue for it https://github.com/Rust-GPU/cargo-gpu/issues/36